src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityFluidDrill.java

```diff
@@ -0,0 +1,305 @@
+        return MetaBlocks.FRAMES.get(Materials.Steel).getBlock(Materials.Steel);
+    }
+
+    @Override
+    public ICubeRenderer getBaseTexture(IMultiblockPart sourcePart) {
+        if (tier == GTValues.MV)
+            return Textures.SOLID_STEEL_CASING;
+        if (tier == GTValues.HV)
+            return Textures.STABLE_TITANIUM_CASING;
+        if (tier == GTValues.EV)
+            return Textures.ROBUST_TUNGSTENSTEEL_CASING;
+        return Textures.SOLID_STEEL_CASING;
+    }
+
+    @Override
+    protected void addDisplayText(List<ITextComponent> textList) {
+        super.addDisplayText(textList);
+        if (!isStructureFormed())
+            return;
+
+        if (energyContainer != null && energyContainer.getEnergyCapacity() > 0) {
+            long maxVoltage = Math.max(energyContainer.getInputVoltage(), energyContainer.getOutputVoltage());
+            String voltageName = GTValues.VNF[GTUtility.getTierByVoltage(maxVoltage)];
+            textList.add(new TextComponentTranslation("gregtech.multiblock.max_energy_per_tick", maxVoltage, voltageName));
+        }
+
+        if (!minerLogic.isWorkingEnabled()) {
+            textList.add(new TextComponentTranslation("gregtech.multiblock.work_paused"));
+
+        } else if (minerLogic.isActive()) {
+            textList.add(new TextComponentTranslation("gregtech.multiblock.running"));
+            textList.add(new TextComponentTranslation("gregtech.multiblock.progress", getProgress() / getMaxProgress() * 100));
+        } else {
+            textList.add(new TextComponentTranslation("gregtech.multiblock.idling"));
+        }
+
+        if (!drainEnergy(true)) {
+            textList.add(new TextComponentTranslation("gregtech.multiblock.not_enough_energy").setStyle(new Style().setColor(TextFormatting.RED)));
+        }
+
+        if (minerLogic.isInventoryFull())
+            textList.add(new TextComponentTranslation("gregtech.multiblock.large_miner.invfull").setStyle(new Style().setColor(TextFormatting.RED)));
+    }
+
+    @Override
+    public void addInformation(ItemStack stack, @Nullable World player, List<String> tooltip, boolean advanced) {
+        super.addInformation(stack, player, tooltip, advanced);
+        tooltip.add(I18n.format("gregtech.machine.fluid_drilling_rig.description"));
+        tooltip.add(I18n.format("gregtech.machine.fluid_drilling_rig.production", getRigMultiplier()));
+        tooltip.add(I18n.format("gregtech.machine.fluid_drilling_rig.energy", GTValues.VNF[tier], GTValues.VNF[tier + 1]));
+    }
+
+    @Override
+    public int getTier() {
+        return this.tier;
+    }
+
+    public int getRigMultiplier() {
+        if (this.tier == GTValues.MV)
+            return 1;
+        if (this.tier == GTValues.HV)
+            return 16;
+        if (this.tier == GTValues.EV)
+            return 64;
+        return 1;
+    }
+
+    public int getDepletionChance() {
```

Comment:
I don't understand. On top of producing more fluid for a higher tier drill, the depletion chance goes down?